packages/manager/modules/pci/src/projects/project/private-networks/vrack/add/add.service.js

```diff
@@ -24,6 +24,16 @@ export default class {
           status: 'done',
         },
       },
-    );
+    )
+      .then(
```

Comment:
```suggestion
      .then(() => true)
      .catch((error) => 
          error.status === 404
              ? true
              : error
      )
```
?

Reply:
Rejection is needed when we have another error than 404 to be catched by transitions hook and go to error page

Reply:
true ! But the calls to `Promise.resolve` could be removed, couldn't they ?